### PR TITLE
Remove User FAQ refer links

### PR DESCRIPTION
**Pull request by kairen**
Fixes issue #5462.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5467)

### CODE REVIEW

File: docs/tasks/debug-application-cluster/troubleshooting.md [hunk 1]
```diff
@@ -33,7 +33,6 @@ and command-line interfaces (CLIs), such as [`kubectl`](/docs/user-guide/kubectl
 
 We also have a number of FAQ pages:
 
-   * [User FAQ](https://github.com/kubernetes/kubernetes/wiki/User-FAQ)
    * [Debugging FAQ](https://github.com/kubernetes/kubernetes/wiki/Debugging-FAQ)
```
Comment (heckj) [inline docs/tasks/debug-application-cluster/troubleshooting.md]: @bgrant0607 question for you - in the work around https://github.com/kubernetes/community/issues/116, are either of the Debugging FAQ or Services FAQ pages due to be removed as well? i.e. https://github.com/kubernetes/kubernetes/wiki/Debugging-FAQ or https://github.com/kubernetes/kubernetes/wiki/Services-FAQ?

Comment (bgrant0607) [reply docs/tasks/debug-application-cluster/troubleshooting.md]: @heckj All of the wiki pages need to be removed eventually. Please extend your issue or file another to ensure that content that is still relevant is moved to actual docs.

Comment (bgrant0607) [reply docs/tasks/debug-application-cluster/troubleshooting.md]: BTW, the community issue refers to community wiki pages. But we also need to get rid of the (even older) kubernetes/kubernetes wiki pages

**Comment by kairen** [comment]
@heckj Refer from @bgrant0607 comments, I removed all of the wiki refer links.

**Pull request merged**